package.json

```diff
@@ -84,7 +86,7 @@
     "typescript": "^4.3.5"
   },
   "engines": {
-    "node": "10.* || 12.* || >= 14.*"
+    "node": "10.* || 12.* || 14.* || 15.* || >= 16.*"
```

Comment:
From what I can tell, this isn't necessary. If you test these ranges on https://semver.npmjs.com/ they do exactly the same thing...

Reply:
So we'd have to update the engines then anyway, right? Why not just leave it until then?